### PR TITLE
Git_Basics.rst: Rename 'coala/coala-bears'

### LINKED ISSUE #4095: Git_Basics.rst: Rename "coala/coala-bears"
`coala/coala-bears` could be mistakenly thought to refer to `https://github.com/coala/coala-bears`.
Change references to the official coala or coala-bears repositories from `coala/coala-bears` to `coala or coala bears.`
difficulty/newcomer

Opened via [gitter](https://gitter.im/coala/coala/?at=58ee2e0d69a692963ea2ece8) by @PierceAndy

### PR DESCRIPTION
**Pull request by acashmkj**
Removing ambiguity between coala or coala-bears and coala/coala-bears
in Git_Basics.rst

Closes https://github.com/coala/coala/issues/4095

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: docs/Developers/Git_Basics.rst [hunk 1]
```diff
@@ -329,7 +329,7 @@ the desired branch:
 
     This assumes that the remote ``upstream`` is the original
     coala repository at https://github.com/coala/coala (or other,
-    like coala/coala-bears, etc.), **not your fork**.
+    like coala or coala-bears, etc.), **not your fork**.
```
Comment (harryjas) [inline docs/Developers/Git_Basics.rst]: This line needs not to be changed.

**Comment by acashmkj** [comment]
@harryjas reverted the changes. Please review

**Comment by acashmkj** [comment]
@prnvdixit but i mentioned closes <issue number>

**Comment by prnvdixit** [comment]
@acashmkj I meant in commit actually :smile:

**Comment by harryjas** [comment]
@acashmkj  Your commit message has more than permitted characters per line. So, please checkout that.

**Comment by acashmkj** [comment]
@harryjas made changes accordingly. Please review

**Comment by harryjas** [comment]
ack

**Comment by harryjas** [comment]
ack ce9ef7a

**Comment by Makman2** [comment]
The issue is still not appearing in the commit. It's only appearing in the PR description, that is something different^^

**Comment by Makman2** [comment]
unack ce9ef7a

**Comment by Makman2** [comment]
Also please squash your commits :+1:

**Comment by acashmkj** [comment]
@Makman2 do you mean Fixes https://github.com/coala/coala/issues/4095 Git_Basics.rst: Rename 'coala/coala-bears' ?

**Comment by Makman2** [comment]
Yeah this should appear in the commit body. Btw, your new commit-body duplicates the sentence in the shortlog ;)

**Comment by Makman2** [comment]
And please use "Closes" instead of "Fixes" :+1:

**Comment by acashmkj** [comment]
Please review

**Comment by prnvdixit** [comment]
@acashmkj Good work... 👍 😄

**Comment by acashmkj** [comment]
@prnvdixit thanks :) . If all the issues are solved can this PR be merged ?

**Comment by prnvdixit** [comment]
@acashmkj Any maintainer would either merge or request changes in the PR soon after reviewing it again.... Changes LGTM. 👍
ack 6f8a47a16925c38d17a2394783dd212992b239af

**Comment by acashmkj** [comment]
@prnvdixit should i ask on gitter for merging this PR ?

**Comment by acashmkj** [comment]
@Makman2 could you merge this PR . Thanks in advance :)

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**